File: Data Structures & Algorithms/top-k-elements-in-list/submission-3.py
from typing import List

class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        counted = {}
        bucket = [[] for _ in range(len (nums)+ 1)]
        for num in nums:
            counted[num] = counted.get(num, 0) + 1

        for key, value in counted.items():
            bucket[value].append(key)

        result = []
        for i in range(len(bucket) - 1, 0, -1):
            for num in bucket[i]:
                result.append(num)
                if len(result) == k:
                    return result

File: Data Structures & Algorithms/sort-colors/submission-3.go
func sortColors(nums []int) {

    buckets := make([]int, 3)

    for _, v := range(nums) {
        buckets[v]++
    }

    i := 0
    for n := range(len(buckets)){
        for range(buckets[n]) {
            nums[i] = n
            i++
        }
    }
    
}
